### DISCUSSION TITLE
这么过分一定要发言（关于Table的rowSelection.preserveSelectedRowKeys）

### DISCUSSION THREAD

**Discussion by taonima** [ant-design/ant-design#28707] [General]
在4.4+之后 新增了preserveSelectedRowKeys属性，  但是更新日志里没对这个属性做说明，并且这个属性为true时操作期望才和4.4-一样，但是这个属性 antd并没有给默认值
不知道有没有同学被这个问题困扰

**Comment by xrkffgg** [comment]
https://github.com/ant-design/ant-design/pull/24718
